Packs/GoogleCloudFunctions/Integrations/GoogleCloudFunctions/GoogleCloudFunctions.py

```diff
@@ -169,15 +171,24 @@ def format_parameters(parameters: str) -> str:
     return json.dumps(dict(pairs))
 
 
+def set_default_region(region):
+    if region is None:
+        # from Google API : If you want to list functions in all locations, use "-" in place of a location
+        return "-"
+    return region
+
+
 def main():
     credentials_json = json.loads(demisto.params().get('credentials_json', {}))
     project = demisto.params().get('project_id', '')
     region = demisto.params().get('region')
+    region = set_default_region(region)
```

Comment:
You can use the following for default values:
```suggestion
    region = demisto.params().get('region', '-')
```
Add the comment above it and youre good

Reply:
I did so but when I ran on Demisto region left None.. So I added this function to be on the safe side